### DISCUSSION TITLE
RegEx filename parser

### DISCUSSION THREAD

**Discussion by mizaki** [comictagger/comictagger#545] [Ideas]
There is `simple` and `complicated` but people might wish to make there own regex to parse file names. Each GenericMetadata field could be supported with a named capturing group `^(?P<title>.*?) #(?P<number>\d+) \((?P<year>\d{4})\)$`

Any objection or further considerations for such a thing?

**Comment by lordwelch** [comment]
I'm not really interested in supporting such a thing, but re-organizing the filename parsers into metadata style or it's own plugin is something that I would do
